packages/grid/src/vaadin-grid.js

```diff
@@ -191,6 +191,7 @@ registerStyles('vaadin-grid', gridStyles, { moduleId: 'vaadin-grid-styles' });
  * `expanded-row-cell`        | Cell in an expanded row
  * `details-opened-row-cell`  | Cell in an row with details open
  * `dragstart-row-cell`       | Cell in a row that user started to drag (set for one frame)
+ * `dragstart-source-row-cell`| Cell in the source row that user started to drag
```

Comment:
The part name (`dragstart...`) and description suggest that the name is only available during the drag start rather than while the rows are dragged. The concept "source row" vs "row" is also confusing.

Reply:
True. I could rename it to `drag-source-row-cell`? Maybe the description could say "cell in the source row of the dragged row"?